package/CHANGELOG

```diff
@@ -44,6 +44,19 @@ Enhancements
    `n_initial_contacts` attribute, with documentation. (Issue #2604, PR #4415)
  * Implement average structures with iterative algorithm from 
    DOI 10.1021/acs.jpcb.7b11988. (Issue #2039, PR #4524)
+ * Removed type and mass guessing from all topology parsers (PR #3753)
+ * Added guess_TopologyAttrs() API to the Universe to handle attribute
+   guessing (pr #3753)
+ * Added the DefaultGuesser class, which is a general-purpose guesser with
+   the same functionalities as the existing guesser.py methods (PR #3753)
+ * Removed type guessing from RDKitParser and assigned type attribute
+   to be equal to the element attribute, if type is not provided (PR #3753)
+ * Added is_value_missing() to `TopologyAttrs` to check for missing
+   values (PR #3753)
+ * Added the `Element` attribute if all names are valid element symbols for
```

Comment:
Just add a comment here as a reminder to check if this is correct (based on my last comment)

Reply:
ok